### ISSUE TITLE
Use GCC 7 on Ubuntu 18

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/OSPSuite.SimModel#144]
Seems that the AppVeyor configuration was upgraded and GCC 9 is used as per default now under Ubuntu, which is not part of 18.04
(so RE builds now fail on Ubuntu, e.g. https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/runs/6224408256?check_suite_focus=true)

**Comment by msevestre** [comment]
What's the problem?

**Comment by Yuri05** [comment]
The problem is: Ubuntu18 build image on AppVeyor comes with different GCC versions: (https://www.appveyor.com/docs/linux-images-software/#compilers).
Ubuntu 18.04 comes with GCC7 out of the box, and it was also the default GCC version on AppVeyor:
![grafik](https://user-images.githubusercontent.com/25061876/165944493-a3f685e5-d319-47ed-bb8e-23cadb6b4bed.png)

But some time ago it apparently changed and they switched to GCC9 as default:
![grafik](https://user-images.githubusercontent.com/25061876/165944644-9c68f9ef-0cb5-4948-b7ce-94e14b3d88bc.png)

Because of that SimModel was linked against newer C++ libs and was not running. 
The solution would be to upgrade Ubuntu to GCC9. This is not quite straight forward but possible (s. e.g. [here](https://gist.github.com/alexandreelise/192a63e287018ddfc896bbcb70b219d4))

However I failed to install it on my Ubuntu18-WSL (and ended up with completely destroyed Linux installation).
So I decided to downgrade to the previous compiler before I invest more time in this.
Unless SimModel is not using the newest C++ features, this is OK.

As soon as we get rid of rClr and can move to newer Ubuntu versions (20.04 at least) - all of that should not be a problem anyway.
